iphone/TitaniumKit/TitaniumKit/Sources/API/TiBlob.m

```diff
@@ -339,6 +339,15 @@ - (void)append:(TiBlob *)blob
   RELEASE_TO_NIL(newData);
 }
 
+- (BOOL)compare:(TiBlob *)blob withTolerence:(CGFloat)tolerence
```

Comment:
It's in our test suite now

Reply:
Okey. I'll remove the native implementation.